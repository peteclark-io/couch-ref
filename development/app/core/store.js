'use strict';

import { createStore } from 'redux';
import reducer from '../ducks/reducers';

const store = createStore(reducer);

export default store;
