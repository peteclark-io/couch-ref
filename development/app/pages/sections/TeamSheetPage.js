'use strict';

import React from 'react';
import LiveTeamSheet from '../../components/Matches/LiveTeamSheet';

const TeamSheetPage = React.createClass({

  render: function() {
    return (
      <LiveTeamSheet id={this.props.params.matchId} />
    );
  }
});

export default TeamSheetPage;
