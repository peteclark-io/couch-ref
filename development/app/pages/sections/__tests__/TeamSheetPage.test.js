'use strict';

import React from 'react';
import TeamSheetPage from '../TeamSheetPage';
import {shallow} from 'enzyme';

import {Provider} from 'react-redux';
import store from '../../../core/store';

test('TeamSheetPage section', () => {
   var params = {matchId: 1001}
   const rendered = shallow(
      <Provider store={store}>
         <TeamSheetPage params={params} />
      </Provider>
   );

   let tree = rendered.debug();
   expect(tree).toMatchSnapshot();
});
