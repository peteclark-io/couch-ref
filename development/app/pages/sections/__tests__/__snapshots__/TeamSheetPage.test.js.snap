exports[`test TeamSheetPage section 1`] = `"<TeamSheetPage params={{...}} />"`;
